examples/openai-agents/tool-calling-agent/tool_calling_agent/util.py

```diff
@@ -0,0 +1,19 @@
+from agents import FunctionTool
+
+def convert_to_openai_agent_tool(spec):
+    spec["function"]["parameters"]["additionalProperties"] = False
```

Comment:
what about adding comments for this magic?

Reply:
about what its purpose is or why this is needed?

Reply:
maybe defining a new function in Spec class to have this addtional_arguments to be stripped?